document/core/exec/numerics.rst

```diff
@@ -135,6 +152,143 @@ When a number is stored into :ref:`memory <syntax-mem>`, it is converted into a
 Again these functions are invertable bijections.
 
 
+.. index:: SIMD, shape
+.. _aux-simdto:
+.. _aux-simdof:
+
+SIMD
+....
+
+SIMD values have the same underlying representation as an |i128|. They can also be interpreted as numeric values packed into a |V128| with a particular |shape|.
+
+.. math::
+   \begin{array}{lll@{\qquad}l}
+   \simdto_{t\K{x}N}(c) &=&
+     c_0~\dots~c_N \\
+     && (\where
+       w = |t| / 8 \wedge
+       b^* = bytes_{\i128}(i) \\ &&
+       \wedge c_j = bytes_{t}^{-1}(b^*[j*w \slice ((j+1)*w)])
+       )
+   \end{array}
+
+..
+   bits_{i8}^{-1}(b^*[0 \slice 8])~
```

Comment:
4. This section is all my previous attempts to define the \simdto function, can be ignored. I kept them around for reference and will remove them before merge (if we are happy with the current definition above).